src/calendar/index.js

```diff
@@ -15,9 +15,8 @@ import BasicDay from './day/basic';
 import Day from './day/index';
 
 //Fallback for react-native-web or when RN version is < 0.44
-const {View, ViewPropTypes} = ReactNative;
-const viewPropTypes =
-  typeof document !== 'undefined' ? PropTypes.shape({style: PropTypes.object}) : ViewPropTypes || View.propTypes;
+const {View, ViewPropTypes, Platform} = ReactNative;
+const viewPropTypes = Platform.OS === "web" ? {style: PropTypes.array} : ViewPropTypes || View.propTypes;
```

Comment:
Why the viewPropTypes for web platform for Calendar is type 'array' and in Agenda is type 'object'? Aren't they both need to be 'array' and 'object' types? Also, if the style is passed from a StyleSheet shouldn't it be type 'number'?

Reply:
Agree it should be object. From what I can tell, StyleSheets used to be created as number, but that is not the case anymore. Return type in [the docs](https://reactnative.dev/docs/stylesheet#create) is object and tracing the definition of ViewPropTypes.style back through the source code also only leads to [some object type](https://github.com/facebook/react-native/blob/6e6443afd04a847ef23fb6254a84e48c70b45896/Libraries/StyleSheet/StyleSheetTypes.js#L555).